extensions/wikia/AdEngine/js/AdEngine2.js

```diff
@@ -25,6 +26,22 @@ define('ext.wikia.adEngine.adEngine', [
 		return func;
 	}
 
+	function prepareAdProviderContainer(providerName, slotName) {
+		var providerContainerId = providerName + '_' + slotName.split('.')[0],
```

Comment:
@gabrys this case is mostly connected with Liftium AFAIASK, right? What do you think about the idea to leave `//TODO: remove after Liftium-era` comment here? ;)